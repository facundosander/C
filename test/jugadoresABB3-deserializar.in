crearPilaJugadores

imprimirCantidadPila

# Construyo Pila

# -- Primer jugador jugador1 20 --
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 20 21 Uruguay jugador1

apilarPilaJugadores
liberarJugador

# -- Segundo jugador jugador2 2 --
crearJugada
1 4 3 2 1
agregarEnPartida
crearJugador 2 19 Brasil jugador2

apilarPilaJugadores
liberarJugador

# -- Tercer jugador jugador3 7 --
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 7 21 Uruguay jugador3

apilarPilaJugadores
liberarJugador

# Transformo lista en arbol
deserializarArbolJugadores

Fin