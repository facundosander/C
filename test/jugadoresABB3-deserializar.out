1>2>La cantidad de jugadores en la pila es: 0.
3>#  Construyo Pila.
4>#  -- Primer jugador jugador1 20 --.
5>6>7>8>9>10>#  -- Segundo jugador jugador2 2 --.
11>12>13>14>15>16>#  -- Tercer jugador jugador3 7 --.
17>18>19>20>21>22>#  Transformo lista en arbol.
23>Jugador 2:  jugador2, Brasil, 19 años
Jugada: 1 
Movimientos: Abajo Arriba Izquierda Derecha 
Jugador 7:  jugador3, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 20:  jugador1, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
24>Fin.
